doc/interfaces/multicast_ip.rst

```diff
@@ -0,0 +1,57 @@
+.. _multicast_ip_doc:
+
+Multicast IP Interface
+======================
+
+This module implements transport of CAN and CAN FD messages over UDP on Multicast IPv4 and IPv6.
+This virtual interface allows to communicate between multiple processes and even hosts.
+This differentiates it from the :ref:`virtual_interface_doc` interface,
+which can only passes messages within a single process but does not require a network stack.
+It runs on UDP to have the lowest possible latency (as opposed to using TCP).
+
+.. note::
+    There are cases in which multicast IP is not an option, e.g. because it is unsupported or
+    blocked intermediate by routers. In this case you can also consider using the related tool
+    `python-can-remote <https://github.com/christiansandberg/python-can-remote>`_.
```

Comment:
I also made an attempt at virtualcan, it can be found here: https://github.com/windelbouwman/virtualcan

It also contains a python-can entry point. Maybe it can be mentioned here as well?

Reply:
Sure, if it helps the users. Could you prepare the wording for it, in particular how it differs from the other virtual interfaces?

Reply:
Notable features:
- TCP/IP connections to a central central server approach
- custom CAN message serialization
- implementations in Python, C++ and rust (for cross language interoperability)
- a server to physical can bus written in rust (for speed reasons)

The reason why I created yet another binary serialization of a CAN message is that I could not get MessagePack to build easily on C++ with windows. This is a bit lame, but the C++ code now is fairly simple, and has few dependencies.

I'm not fully aware how the other virtual interfaces function, but I'm guessing it's more of the same?

Reply:
That should suffice. Its somewhat different, the included one is internally just a global list that relays it within the same process. Dead simple. ;)